core/types/header_ext_test.go

```diff
@@ -164,7 +163,7 @@ func allFieldsSet[T interface {
 				assertNonZero(t, f)
 			case *Header:
 				assertNonZero(t, f)
-			case []uint8, []*Header, Transactions, []*Transaction, ethtypes.Withdrawals, []*ethtypes.Withdrawal:
+			case []uint8, []*Header, Transactions, []*Transaction, Withdrawals, []*Withdrawal:
```

Comment:
Should we import from libevm for coreth test files in this package, and keep imports_test.go reserved for avalanchego tests?

Reply:
I'm not sure what version you reviewed that had the `ethtypes.` in there because I'm seeing it without.

Once AvalancheGo is updated to use the `libevm` types instead of `coreth` then we can do a refactor here to remove all aliasing (i.e. delete `avalanchego.go` and `imports_test.go`). This will force all tests to use `libevm/core/types` so we'll have no choice; I'd be open to dot-importing it in these tests to avoid hundreds of changes.

Reply:
They've [been removed](https://github.com/ava-labs/coreth/pull/875#discussion_r1995903235).